manifests/install.pp

```diff
@@ -6,13 +6,25 @@
 ) {
   include ::nunaliit::params
 
+
   # Install nunaliit tgz from source url
-  $nunaliit_tar_url = "http://central.maven.org/maven2/ca/carleton/gcrc/nunaliit2-couch-sdk/${nunaliit_version}/nunaliit2-couch-sdk-${nunaliit_version}-nunaliit.tar.gz"
-  puppi::netinstall { "install-nunaliit-${nunaliit_version}":
-    url             => $nunaliit_tar_url,
-    destination_dir => '/opt',
-    extracted_dir   => "nunaliit2-couch-sdk-${nunaliit_version}"
-  }
 
+  if $nunaliit_version =~ /.*SNAPSHOT.*/ {
+    $nunaliit_tar_path = "/apps/nunaliit/distros/nunaliit2-couch-sdk-${nunaliit_version}.tar.gz"
```

Comment:
Where is this path from? Is this a path that would have to be set to a download location? If so, can it be put in the nunaliit params so it can be overridden?

Reply:
This is work in progress right now. It is bullet 7 in your task. This was a first pass and Peter needs to get more details. Putting the path somewhere it can be changed makes a lot of sense. I'm not sure where that path should go that it can be updated later but that it doesn't confuse less savy users.

Reply:
OK. Since it won't effect normal use I don't see a reason not to merge it now the way it is then with the understanding that where to put that path is part of what needs to be worked out. Can you put a comment in to remind us that this is in the works?

Reply:
Just committed a couple TODO comments so this doesn't get forgotten. Let me know if you want me to add anything else.